MVCapplication/src/main/webapp/WEB-INF/views/categories.jsp

```diff
@@ -12,4 +13,32 @@
 </div>
 
 <!-- Table with the list of categories -->
+<div class="row">
+    <div class="col-lg-12">
+        <div class="panel panel-default">
+            <%--<div class="panel-body">--%>
 
+                <table width="100%" class="table table-striped table-bordered table-hover">
+                    <thead>
+                    <tr>
+                        <th class="text-center">ID</th>
+                        <th class="text-center">Name</th>
+                        <th class="text-center">Edit</th>
+                        <th class="text-center">Delete</th>
+                    </tr>
+                    </thead>
+                    <tbody>
+                    <c:forEach items="${categories}" var="category">
+                        <tr class="gradeA">
+                            <td><c:out value="${category.id}"></c:out></td>
+                            <td><c:out value="${category.name}"></c:out></td>
```

Comment:
I mean that 
` <td><c:out value="${category.name}"></c:out></td>`
and 
` <td>${category.name}"></td>`
doing the same thing in this case